README.md

```diff
@@ -1,2 +1,4 @@
 # newproject
+
 my mini game
+"This game is about collecting gold coins."
```

Comment:
Is this the complete thing??

Reply:
[2f5208d](https://github.com/jumjumshaik/newproject/pull/1/commits/2f5208d81c23fc5b5157118fb488a522fe28758c)

updated